server/connector/connector.go

```diff
@@ -194,22 +272,45 @@ func (c *connector) Post(w http.ResponseWriter, req *http.Request) {
 		} else {
 			http.Error(w, fmt.Sprintf(`{"error":"unknown error: %s"}`, err.Error()), http.StatusInternalServerError)
 		}
+
 		return
 	}
 	go c.Run(subscriber)
 	c.logger.WithField("topic", topic).Info("Subscription created")
 	fmt.Fprintf(w, `{"subscribed":"/%v"}`, topic)
+
+	if errFill == nil {
+		err = event.report(c.KafkaProducer, c.KafkaReportingTopic)
```

Comment:
if the Producer is nil or the reporting topic empty, we will have a log error for every event/message.
maybe we can check these 2 parameters when the connector is created, and log a single error message there ?